File: PatientService/PatientService/PatientService.Db/Entities/Address.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace PatientService.Db.Entities
{
    public class Address
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public string Country { get; set; }
        [Required]
        public string State { get; set; }
        [Required]
        public string City { get; set; }
        [Required]
        public string ZipCode { get; set; }
        public bool IsPrimary { get; set; }

        [ForeignKey(nameof(Patient))]
        public int PatientId { get; set; }
        public virtual Patient Patient { get; set; }
    }
}

File: PatientService/PatientService/PatientService.WebClient/ClientApp/src/app/address/Address.ts
import {Patient} from "../patient/Patient";

export class Address {
  constructor(
    public id?: number,
    public country?: string,
    public state?:  string,
    public city?: string,
    public zipCode?: string,
    public isPrimary?: boolean,
    public patientId?: number,
    public patient?: Patient
  ) {}
}
